Cannot pass compilation under macos

I have been following the project for a while, it worked fine until this commit: https://github.com/RustPython/RustPython/commit/23cba402a5ea03d4fb2f5f6f22d8e9e35c67645d
now with macos, the project cannot compile correctly.
```bash
error[E0425]: cannot find value `SIGSTKFLT` in module `libc`
   --> vm/src/stdlib/signal.rs:141:42
    |
141 |         "SIGSTKFLT" => ctx.new_int(libc::SIGSTKFLT as u8),
    |                                          ^^^^^^^^^ help: a constant with a similar name exists: `SIGSTKSZ`

error[E0425]: cannot find value `SIGPWR` in module `libc`
   --> vm/src/stdlib/signal.rs:155:39
    |
155 |         "SIGPWR" => ctx.new_int(libc::SIGPWR as u8),
    |                                       ^^^^^^ not found in `libc`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0425`.
error: Could not compile `rustpython-vm`.

To learn more, run the command again with --verbose.
```

and  @coolreader18 suggested have a look at `#[cfg]`s for wasm for `signal` and add macos to have a workaround.

--- a/vm/src/stdlib/signal.rs
+++ b/vm/src/stdlib/signal.rs
@@ -138,7 +138,6 @@ fn extend_module_platform_specific(vm: &VirtualMachine, module: PyObjectRef) -> 
         "SIGUSR2" => ctx.new_int(libc::SIGUSR2 as u8),
         "SIGPIPE" => ctx.new_int(libc::SIGPIPE as u8),
         "SIGALRM" => ctx.new_int(libc::SIGALRM as u8),
-        "SIGSTKFLT" => ctx.new_int(libc::SIGSTKFLT as u8),
         "SIGCHLD" => ctx.new_int(libc::SIGCHLD as u8),
         "SIGCONT" => ctx.new_int(libc::SIGCONT as u8),
         "SIGSTOP" => ctx.new_int(libc::SIGSTOP as u8),
@@ -152,9 +151,16 @@ fn extend_module_platform_specific(vm: &VirtualMachine, module: PyObjectRef) -> 
         "SIGPROF" => ctx.new_int(libc::SIGPROF as u8),
         "SIGWINCH" => ctx.new_int(libc::SIGWINCH as u8),
         "SIGIO" => ctx.new_int(libc::SIGIO as u8),
-        "SIGPWR" => ctx.new_int(libc::SIGPWR as u8),
         "SIGSYS" => ctx.new_int(libc::SIGSYS as u8),
     });
+
+    #[cfg(not(target_os = "macos"))]
+    {
+        extend_module!(vm, module, {
+            "SIGPWR" => ctx.new_int(libc::SIGPWR as u8),
+            "SIGSTKFLT" => ctx.new_int(libc::SIGSTKFLT as u8),
+        });
+    }
 
     module
 }

